crates/sui-transaction-builder/src/lib.rs

```diff
@@ -964,7 +919,7 @@ mod tests {
                             tx.input(unresolved::Input::immutable(o, obj.version(), obj.digest())));
                         }
                         sui_types::types::Owner::Shared(x) => {
-                            upgrade_cap = Some(tx.input(unresolved::Input::shared(o, *x, Some(true))));
+                            upgrade_cap = Some(tx.input(unresolved::Input::shared(o, *x, true)));
```

Comment:
Can you use `tx.input(&o)` here too now?

Reply:
Yesss 🕺